src/main/java/com/alphasegroup/attendancetracker/Controllers/ClassMeetingController.java

```diff
@@ -49,10 +51,12 @@ public String generateClassMeeting(@RequestParam(name="sectionid", required=true
 		String uri = "https://api.qrserver.com/v1/create-qr-code/?" +
 					"size=200x200&data=www.google.com";
 		// URI fin = null;
-		// try {
-		// 	fin = new URI(uri);
-		// }
-		// catch(Exception e) {}
+		//try {
+		//	fin = new URI(uri);
+		//}
+		//catch(Exception e) {}
+		SpringApplication.run(qrCodeService.class, uri);
```

Comment:
What does this line do? It doesn't return the http response from the service, so I'm just curious why it's in here.

Reply:
It's left-over from when I thought I could make this do more than it does.